build-system/tasks/bundle-size/README.md

```diff
@@ -0,0 +1,19 @@
+# Bundle Size checks
+
+The `APPROVERS.json` file is a listing of compiled AMP runtime and extensions
+files, and for each file what teams are allowed to approve pull requests that
+increase to that file's bundle size (Brotli-compressed size) and what is the
+threshold of an increase that requires special approval, in KBs.
```

Comment:
This is a long sentence to parse. It might be clearer so say something like 

```markdown
The `APPROVERS.json` file is a listing of compiled AMP runtime and extensions files. Each rule has two parts:
- `approvers`:  blah blah blah
- `threshold`: bloo bloo bloo
```